server/database/quieres/teacherProfile.js

```diff
@@ -0,0 +1,7 @@
+const connection = require('../config/connection');
+
+const getTeacherProfile = () => connection.query(
+  'select first_name, last_name, bio, image_url from teacher inner join "user" on teacher.user_id="user".id ',
```

Comment:
related to my previous comment, if this is supposed to return just one specific user, you need to parametrize the query
```
... on teacher.user_id="user".id WHERE "user".id = $1',
// pass the id as the parameter
```

Reply:
Done